notebook/static/notebook/js/notebook.js

```diff
@@ -2055,11 +2082,11 @@ define(function (require) {
      * @param {integer} end - index of the last cell to execute (exclusive)
      */
     Notebook.prototype.execute_cell_range = function (start, end) {
-        this.command_mode();
```

Comment:
OK, as long as there are not code paths where that can be called twice accidentally.